.gitignore

```diff
@@ -140,3 +140,4 @@ src/build/
 .azureml/
 samples/script_file.py
 samples/outputs/
+outputs/*
```

Comment:
Is this outputs folder intended?

Reply:
yes, can appear in multiple directories (also test, etc)